config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
   devise_for :partners, controllers: { sessions: "partners/sessions" }
-  devise_scope :partner do
-    get "/partners/sign_out" => "devise/sessions#destroy"
+  devise_scope :partner do # , :skip => [:registrations] # do we need this?
+    get "/partners/:id/sign_out" => "devise/sessions#destroy"
```

Comment:
Why was the route here changed to include the :id ?

Reply:
@nihonjinrxs And I decided to put :id there because we had a sign out error in the app and we assume that it should be signing off from a certain partner's account. But, I'm not 100% sure if it should be there since everything is still new to me and devise is a bit confusing as well.